Gemfile

```diff
@@ -8,6 +8,7 @@ gem "auth0", "~> 4.9"
 gem "bootsnap", ">= 1.1.0", require: false
 gem "bootstrap", ">= 4.3.1"
 gem "coffee-rails", "~> 5.0"
+gem "govuk_design_system_formbuilder", github: "DFE-Digital/govuk_design_system_formbuilder", branch: "i18n"
```

Comment:
Depending on a branch rather than a version seems like a risk. Have we thought about the impact to us if that branch disappears? Are we happy with it? Have we read anything that gives us confidence that this is likely to be merged into a new version?